Pa3/Durchführung/Project/app/State/InitializeMcu.c

```diff
@@ -36,11 +40,28 @@
 
 EventEnum InitializeMcu_InitializeAll(void)
 {
-
+  //HAL
+  if (Gpio_init(void) != 0)
+  {
+    ErrorHandler_halt(ERRORHANDLER_STARTUP_INIT_FAIL);
+  }
+  Pwm_init(void);
+  Tick_Timer_init(void);
+  Irq_init(void);
+  //Service
+  Button_init(void);
+  LineSensor_init(void);
+  Display_init(void);
+  Buzzer_init(void);
+  DriveControl_init(void);
+  LED_init(void);
 } 
 
 void InitializeMcu_DisplayTeamName(void)
 {
-  
+  Display_clear(void);
```

Comment:
on function call the void is not necessary, only in declarations and definitions
```suggestion
  Display_clear();
```